src/content/advanced/spot-instances/azure/ondemand-fallback/index.md

```diff
@@ -0,0 +1,173 @@
+---
+linkTitle: Using on-demand as fall-back on Azure
+title: Using on-demand instances as fall-back when spot instances are unavailable on Azure
+description: When using spot instances in a node pool on Azure, it can happen that the node pool cannot be scaled up as not enough spot instances are available. This guide shows you how to configure cluster-autoscaler in a way to provide on-demand instances as a back-up automatically.
+weight: 30
+menu:
+  main:
+    parent: advanced-spotinstances-azure
+user_questions:
+  - How can I use VM spot instances and fall back to on-demand if no spot instances are available?
+aliases:
+  - /advanced/spot-instances/azure/on-demand-fallback/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-celestial
+---
+
+# Using on-demand instances as fall-back when spot instances are unavailable
+
+As of workload cluster release v{{% first_azure_spotinstances_version %}} for Azure, node pools can use [spot instances]({{< relref "/advanced/spot-instances/azure/overview" >}}). Users can select for each node pool which percentage of nodes should be covered by spot instances and by on-demand instances respectively. And users can define a base amount of on-demand instances that will be used, to ensure at least a certain amount of worker nodes are available at any time.
```

Comment:
But IIUC it is not supported to have mixed Node Pools having on-demand and spot instances at the same time?

Reply:
ah true, forgot to straight that out

Reply:
fixed

Reply:
But what's the configuration called when you set max price to `-1`? Doesn't that behave like a mixed one, even though it's not necessarily explicitly such?

Reply:
u can't set -1 :D

Reply:
_Goes to read docs..._ :joy: 

Why?! [Azure docs](https://docs.microsoft.com/en-us/azure/virtual-machines/spot-vms#eviction-policy) specifically state this:
> If the max price is set to -1: The VM will not be evicted for pricing reasons. The max price will be the current price, up to the price for standard VMs. You will never be charged above the standard price.

For behavior that would be explicitly the desired outcome of "mixed" type node pool. I feel like having to put some arbitrary $highnumber is very confusing if not outright scary :man_shrugging:

Reply:
isn't it this setting: "Use current on-demand pricing as max"? :D i will make docs more specific about that

```"priority": "Spot",
                    "evictionPolicy": "Delete",
                    "billingProfile": {
                        "maxPrice": -1
                    }```

Reply:
Done